docs/advanced/api-v2/rest.md

```diff
@@ -0,0 +1,4019 @@
+                    "block_index": 836519,
+                    "source": "18szqTVJUWwYrtRHq98Wn4DhCGGiy3jZ87",
+                    "destination": "1HC2q92SfH1ZHzS4CrDwp6KAipV4FqUL4T",
+                    "flags": 3,
+                    "status": "valid",
+                    "memo": null,
+                    "fee_paid": 1400000
+                }
+            ]
+        }
+    ```
+
+## Group Transactions
+
+### Info [GET `/transactions/info`]
+
+Returns Counterparty information from a raw transaction in hex format.
+
++ Parameters
+    + rawtransaction: `01000000017828697743c03aef6a3a8ba54b22bf579ffcab8161faf20e7b20c4ecd75cc986010000006b483045022100d1bd0531bb1ed2dd2cbf77d6933273e792a3dbfa84327d419169850ddd5976f502205d1ab0f7bcbf1a0cc183f0520c9aa8f711d41cb790c0c4ac39da6da4a093d798012103d3b1f711e907acb556e239f6cafb6a4f7fe40d8dd809b0e06e739c2afd73f202ffffffff0200000000000000004d6a4bf29880b93b0711524c7ef9c76835752088db8bd4113a3daf41fc45ffdc8867ebdbf26817fae377696f36790e52f51005806e9399a427172fedf348cf798ed86e548002ee96909eef0775ec3c2b0100000000001976a91443434cf159cc585fbd74daa9c4b833235b19761b88ac00000000` (str, required) - Raw transaction in hex format 
+    + block_index (int, optional) - Block index mandatory for transactions before block 335000
+        + Default: `None`
+
++ Response 200 (application/json)
+
+    ```
+        {
+            "result": {
+                "source": "178etygrwEeeyQso9we85rUqYZbkiqzL4A",
+                "destination": "",
+                "btc_amount": 0,
+                "fee": 56565,
+                "data": "16010b9142801429a60000000000000001000000554e4e45474f544941424c45205745204d555354204245434f4d4520554e4e45474f544941424c4520574520415245",
+                "unpacked_data": {
+                    "message_type": "issuance",
+                    "message_type_id": 22,
+                    "message_data": {
+                        "asset_id": 75313533584419238,
+                        "asset": "UNNEGOTIABLE",
+                        "subasset_longname": null,
+                        "quantity": 1,
+                        "divisible": false,
+                        "lock": false,
+                        "reset": false,
+                        "callable": false,
+                        "call_date": 0,
+                        "call_price": 0.0,
+                        "description": "UNNEGOTIABLE WE MUST BECOME UNNEGOTIABLE WE ARE",
+                        "status": "valid"
+                    }
+                }
+            }
+        }
+    ```
+
+### Unpack [GET `/transactions/unpack`]
+
+Unpacks Counterparty data in hex format and returns the message type and data.
+
++ Parameters
+    + datahex: `16010b9142801429a60000000000000001000000554e4e45474f544941424c45205745204d555354204245434f4d4520554e4e45474f544941424c4520574520415245` (str, required) - Data in hex format 
+    + block_index (int, optional) - Block index of the transaction containing this data
```

Comment:
why is this here? shouldn't be any faster...

Reply:
This is to manage protocol changes.. if you want to unpack a transaction before a given change..